src/Form/Options.php

```diff
@@ -87,7 +87,7 @@ protected static function data($model): array
 
 		// add the model by the proper alias
 		foreach (static::aliases() as $className => $alias) {
-			if (is_a($model, $className) === true) {
+			if ($model instanceof $className) {
```

Comment:
I just really wonder that does `instanceof` also work correctly when  the comparator (`$className`) is string? 🤯

Reply:
I thought first it would not, which is why I left it out. But it seems to actually work.

Reply:
Thanks! Now I tried different combinations. It does indeed seem to be working correctly 👍

Reply:
Works since PHP 8.0.